apps/betterangels/src/app/(private-screens)/add-note/Location/LocationMapModal/Header.tsx

```diff
@@ -32,13 +32,12 @@ export default function Header(props: IHeaderProps) {
       </Pressable>
       <TextBold
         style={{
-          flex: 2,
           textAlign: 'center',
         }}
       >
         Type or Pin Location
       </TextBold>
-      <View style={{ flex: 1 }} />
+      <View style={{ width: 34.5, height: 1 }} />
```

Comment:
this is a very specific width. how will this affect other devices?

Reply:
same width as back. 
"back" will be same width everywhere